### PR TITLE
Correction of getpetinfo parameter in item db

### PR DESCRIPTION
**Pull request by Atemo**
* **Addressed Issue(s)**: https://github.com/rathena/rathena/issues/7398

* **Server Mode**: renewal

* **Description of Pull Request**: 

The parameter required is `PETINFO_CLASS` (mob ID) instead of `PETINFO_ID` (pet unique ID)

Thanks to @kaninhot004

### CODE REVIEW

**Comment by idk-whoami** [comment]
I think should use getpetinfo(PETINFO_EGGID)  :)

**Comment by idk-whoami** [comment]
It works.

File: db/re/item_db_equip.yml [hunk 1]
```diff
@@ -143168,29 +143168,29 @@ Body:
     Script: |
       bonus2 bAddSize,Size_All,10;
       bonus2 bMagicAddSize,Size_All,10;
-      if (getpetinfo(PETINFO_ID) == 9112)
+      if (getpetinfo(PETINFO_EGGID) == 9112)
```
Comment (Lemongrass3110) [inline db/re/item_db_equip.yml:143171]: Change it into a switch?

File: db/re/item_db_equip.yml [hunk 2]
```diff
@@ -143205,19 +143205,19 @@ Body:
     EquipLevelMin: 100
     Script: |
       bonus bAspdRate,10;
-      if (getpetinfo(PETINFO_ID) == 9109 || getpetinfo(PETINFO_ID) == 9112 || getpetinfo(PETINFO_ID) == 9115 || getpetinfo(PETINFO_ID) == 9121) {
+      if (getpetinfo(PETINFO_EGGID) == 9109 || getpetinfo(PETINFO_EGGID) == 9112 || getpetinfo(PETINFO_EGGID) == 9115 || getpetinfo(PETINFO_EGGID) == 9121) {
          bonus bBaseAtk,200;
          bonus bMatk,200;
          bonus bAllStats,10;
       }
       /*Todo Itemid 9126 9136*/
```
Comment (Lemongrass3110) [inline db/re/item_db_equip.yml:143213]: The checks could already be added, even though they will never succeed - to get rid of the TODO.

File: db/re/item_db_equip.yml [hunk 3]
```diff
@@ -143205,19 +143205,19 @@ Body:
     EquipLevelMin: 100
     Script: |
       bonus bAspdRate,10;
-      if (getpetinfo(PETINFO_ID) == 9109 || getpetinfo(PETINFO_ID) == 9112 || getpetinfo(PETINFO_ID) == 9115 || getpetinfo(PETINFO_ID) == 9121) {
+      if (getpetinfo(PETINFO_EGGID) == 9109 || getpetinfo(PETINFO_EGGID) == 9112 || getpetinfo(PETINFO_EGGID) == 9115 || getpetinfo(PETINFO_EGGID) == 9121) {
```
Comment (Lemongrass3110) [inline db/re/item_db_equip.yml:143208]: Change it into a switch?

**Review by Lemongrass3110 — APPROVED** [review]
General remark:
Why do we check for the pet egg id, when the text actually refers to the pet monster's name (and therefore mob id) instead of the egg?

**Review by Lemongrass3110 — APPROVED** [review]

**Pull request merged**